CppCoreGuidelines.md

```diff
@@ -6389,7 +6389,7 @@ A `swap` can be handy for implementing a number of idioms, from smoothly moving
         int m2;
     };
 
-Providing a nonmember `swap` function in the same namespace as your type for callers' convenience.
+Providing a non-member `swap` function in the same namespace as your type for callers' convenience.
```

Comment:
M-W doesn't hyphenate https://m-w.com/dictionary/nonmember 
but the standard had a change of heart and switched to "non-member" in 2018 https://github.com/cplusplus/draft/commit/3febb8b9dfe0a9e83a912c6f5fb56687b528261d 
I suppose it's good to be consistent with it here.

Reply:
Thank you for the feedback!
As you mentioned "non-member" looks better option to me as well because the standard is using "non-member" 

(but by the [rule](https://github.com/cplusplus/draft/wiki/Specification-Style-Guidelines#hyphens-nonx-vs-non-x) "nonmember" looks actually correct because most of dictionaries have a word "nonmember")

#### nonmember

* In the stdandard (N4842)
  * 189 times = non-member 101 + nonmember 88 (not in plain text)
* In dictionaries
  * Google : O
  * Oxford : X
  * Collins : O
  * Dictionary.com : O
  * Merriam-Webster : O
  * Macmillan : O
  * Cambridge : O
  * Longman : O

Reply:
I resolve this conversation
If you have any suggestion please let me know !